packages/suite-native-navigation-root/src/dummyScreens/DemoScreen.tsx

```diff
@@ -106,9 +107,9 @@ export const DemoScreen = () => {
                     <Box marginTop="medium">
                         <Select
                             items={selectItems}
-                            selectedItem={selectedItem}
+                            value={selectedItem ?? null}
```

Comment:
why do you need `?? null`

Reply:
Removed and rebased so the commit link won't work since it was fixup commit :( 

You can see I removed it from DemoScreen.tsx